plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/glue/GlueHiveMetastore.java

```diff
@@ -929,10 +931,12 @@ public void addPartitions(HiveIdentity identity, String databaseName, String tab
                     }
                 }
 
-                for (PartitionWithStatistics partition : partitions) {
-                    // TODO(https://github.com/trinodb/trino/issues/7033) make updates in batch
```

Comment:
There isn't a batch API for updating statistics, unfortunately

Reply:
Can we close the issue then with this comment added there?